README.md

```diff
@@ -178,10 +181,18 @@ An array of suggestions that are used as basis for showing suggestions. These ob
 // With labelField of `name`
 const suggestions = [
     { id: "1", name: "mango" },
-    { id: "2", name: "pineapple" },
+    { id: "2", name: "pineapple"},
     { id: "3", name: "orange" },
     { id: "4", name: "pear" }
 ];
+
+//With className to style the tags
```

Comment:
space

Reply:
close the pre tag here and  add a line `If the suggestions have className, the following would work:` here before adding the example.

Reply:
@harshithmullapudi the classnames will only be added in tags and not in suggestions, can you verify this ? in that case lines `189-195` should be removed.